cmd/flagger/main.go

```diff
@@ -65,6 +67,8 @@ var (
 
 func init() {
 	flag.StringVar(&kubeconfig, "kubeconfig", "", "Path to a kubeconfig. Only required if out-of-cluster.")
+	flag.Float64Var(&kubeconfigQPS, "kubeconfig-qps", 0, "Set QPS for kubeconfig.")
```

Comment:
This should be `IntVal`

Reply:
Hey, I just followed here: https://godoc.org/k8s.io/client-go/rest#Config 
it's defined as `float32` but flags just have `float64`

Please confirm to me if I should make the change or not.

Reply:
Please use int

Reply:
done in: 820411f8ec245575a5e89c7545ec0ed8159ff69a